truss-chains/truss_chains/deploy.py

```diff
@@ -160,6 +204,13 @@ def __init__(self, entrypoint: str, name: str) -> None:
     def add_service(self, name: str, service: b10_service.TrussService) -> None:
         self._services[name] = service
 
+    def set_remote_chain_service(
```

Comment:
You brought it already up in the docs-PR, now I have even more mixed feelings about this class and the abstractions / contracts we have between this class and the other ones.

Sooner or later we must improve this. If you don't have an idea or time to do it now, please leave a TODO that this is not an ideal interface and needs to be reworked.